testdata/expected/src/github.com/sgtest/minimal-go-stdlib/dummy1/GoPackage.graph.json

```diff
@@ -62,7 +62,6 @@
       "End": 27
     },
     {
-      "DefRepo": "github.com/golang/go",
```

Comment:
I think this line is removed because now this ref is resolving to a def in the same repo. Earlier it was resolving to the actual Go stdlib repo so it was wrong earlier since this testcase is a dummy stdlib repo itself and all builtin refs should be resolved locally. For non-stdlib repos, it will still resolve correctly, as seen in other testcases, eg https://github.com/sourcegraph/srclib-go/blob/builtin-types/testdata/expected/src/github.com/sgtest/go-misc/github.com/sgtest/go-misc/scope/GoPackage.graph.json#L547-L575